sdks/go/pkg/beam/runners/dataflow/dataflow.go

```diff
@@ -77,8 +77,7 @@ var (
 	teardownPolicy = flag.String("teardown_policy", "", "Job teardown policy (internal only).")
 
 	// SDK options
-	cpuProfiling     = flag.String("cpu_profiling", "", "Job records CPU profiles to this GCS location (optional)")
-	sessionRecording = flag.String("session_recording", "", "Job records session transcripts")
```

Comment:
We'll have to double-check this, but the sessionRecording flag currently does nothing (hence the removed empty branch and removed flag) so I removed it. But it is, technically, a breaking change since someone could be running dataflow jobs with this flag set even though it doesn't do anything.

Reply:
Technically it's likely to fix the job unless it's been very small (outside of breaking the flag being present).

The session runner's job was an honest hearted attempt to make SDK development and bug finding easier by allowing playback of recorded job features in the Early Portability age. These days, it's better if we have clearer tools for that.